controller/templates/controller/partials/state_machine.html

```diff
@@ -0,0 +1,10 @@
+{% load render_table from django_tables2 %}
+{% for event in events %}<input type="hidden" name="event" id="event_{{ event }}" value="{{ event }}">{% endfor %}
+<form>
+  {% csrf_token %}
+  <input type="hidden"
+         name="current_state"
+         id="current_state"
+         value="{{ current_state }}">
```

Comment:
Maybe also add a comment here that this can be removed later.

Reply:
Removed